core/src/main/java/org/apache/iceberg/avro/AvroSchemaUtil.java

```diff
@@ -33,19 +33,21 @@
 import org.apache.iceberg.types.TypeUtil;
 import org.apache.iceberg.types.Types;
 
-import static org.apache.avro.Schema.Type.ARRAY;
-import static org.apache.avro.Schema.Type.MAP;
-import static org.apache.avro.Schema.Type.RECORD;
-import static org.apache.avro.Schema.Type.UNION;
-
 public class AvroSchemaUtil {
+
+  private AvroSchemaUtil() {}
+
   public static final String FIELD_ID_PROP = "field-id";
   public static final String KEY_ID_PROP = "key-id";
   public static final String VALUE_ID_PROP = "value-id";
   public static final String ELEMENT_ID_PROP = "element-id";
   public static final String ADJUST_TO_UTC_PROP = "adjust-to-utc";
 
   private static final Schema NULL = Schema.create(Schema.Type.NULL);
+  private static final Schema.Type MAP = Schema.Type.MAP;
```

Comment:
This looks a lot like a static import to me... Is this just a work-around to avoid too many changes here?

Reply:
Something to that effect yeah. Should Avro types be added to static import exclusions?

Reply:
Sounds reasonable to me. I'm okay either way. I don't really mind static imports in some cases, but it's best to be consistent.